src/common/lib/utils.ts

```diff
@@ -102,7 +102,7 @@ const utils = {
   openPrompt: <Type>(message: {
     args: Record<string, unknown>;
     origin: OriginData | OriginDataInternal;
-    action: string;
+    action: string; // TODO: here action should always match a route PATH in app/router/routes
```

Comment:
I.e.:
```ts
  makeInvoice: "makeInvoice",
  LNURLChannel: "LNURLChannel",
  lnurlWithdraw: "lnurlWithdraw",
  confirmKeysend: "confirmKeysend",
  confirmPayment: "confirmPayment",
  confirmSignMessage: "confirmSignMessage",
```
But should be added step by step with further PRs, right?

Reply:
not sure what this means. why that mapping? 

(btw. we should unify the names `LNURLChannel` vs. `lnurlWithdraw`)

Reply:
As discussed. Not that list but only further LNURL-calls with nw PRs:

- lnurlAuth
- lnurlWithdraw
- LNURLChannel (rename this one to match casing)

To only "support" these first we can trigger the prompt-action from the Send-screen first. We can then refactor and make it feel more integrated (with the usual popup UX) later. We will do this with `lnurlAuth`

Reply:
This is just for TypeScript.

The `action` in util `openPrompt`  is always matching a specific route name. 
Otherwise it would not work.

This is just the beginning of typing the route names [here for lnurpay](https://github.com/getAlby/lightning-browser-extension/pull/1257/files#diff-6d91cb5155c78128e7bcef234215937c2c128d06e51d23fff892313f968fd9f7R3) and the types [here](https://github.com/getAlby/lightning-browser-extension/pull/1257/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR127) and [here](https://github.com/getAlby/lightning-browser-extension/pull/1257/files#diff-6d91cb5155c78128e7bcef234215937c2c128d06e51d23fff892313f968fd9f7R3)

Reply:
To sum up:

- remove the comment